angular-link-diver/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
I think the directory "angular-link-diver" is not needed here since this repository will only contain one project and one buildable unit.

So the directory structure will look like this

/ (root)
/.editorconfig
/.gitignore
/e2e
/docs
/src
....

What do you think?

Reply:
Sure, if I change this now is it going to play nice with the changes in the other branch or should I wait until later to do this?

Reply:
I imagine there will be a conflict in another branch, but we can resolve that conflict quite easily. So we don't need to wait for later.